.circleci/config.yml

```
@@ -168,7 +172,7 @@ workflows:
           cron: "0 0 * * 1-5"
           filters:
             branches:
-              only:
+             only:
```

Bit nitpicky I guess, but, any reason for the indentation change here?
```suggestion
              only:
```